tests/foreman/ui/test_host.py

```diff
@@ -2234,6 +2234,69 @@ def test_positive_read_details_page_from_new_ui(
         assert values['Overview']['DetailsCard']['details']['comment'] == 'Host with fake data'
 
 
+@pytest.mark.tier4
+@pytest.mark.stubbed
+def test_positive_update_delete_package():
+    """Update package on host using the new Content tab
+
+    :id: ffc19a40-85f4-4894-a18b-f6d88b2ce377
+
+    :steps:
+        1. Install a package on registered host.
+        2. Navigate to Content tab.
+        3. Check is package is in upgradable state.
```

Comment:
`Check is package` makes no sense, please fix the sentence. 
Don't know how much we are playing for an English, but  you are missing a lot of articles (a/the), if you don't mind copy the text to Grammarly and you can fix it in no time :).